backend/open_webui/utils/redis.py

```
@@ -1,7 +1,6 @@
 import socketio
-import redis
-from redis import asyncio as aioredis
 from urllib.parse import urlparse
+from typing import Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)